include/service/entities/site_service.js

```diff
@@ -289,6 +289,9 @@ module.exports = function SiteServiceModule(pb) {
      */
     SiteService.prototype.editSite = function(options, cb) {
         cb = cb || util.cb;
+        if (!options.prevHostnames) {
+            options.prevHostnames = [];
+        }
```

Comment:
@brianhyder Er - don't merge yet please. This check and assignment does nothing. My tests were only working on sites with a prevHostnames array pre-defined. Your previous comment suggested adding this assignment to the SiteService.format method but this does not exist, nor should it? I don't want to register for the BaseObjectService's format event in the SiteService which inherits from the BOS right? I don't want to handle events emitted by myself? Would you mind clarifying a little?